locales/en/kafka.json

```diff
@@ -4,10 +4,29 @@
   "clientID": "Client ID",
   "connection_rate": "Connection rate",
   "connection_rate_value": "up to 100 connections/second",
+  "connection_tab": {
+    "access_tab": "Access tab.",
+    "alter_allow": "The owner, the organization administrator, or users with the Allow Alter permission on this instance can manage access.",
+    "authentication_method": "Authentication method",
+    "bootstrap_server": "Bootstrap server",
+    "bootstrap_server_description": "Establish the initial connection between your client and Kafka instance.",
+    "create_service_account_to_generate_credentials": "Create a service account to generate credentials. Manage the service accounts you create on the <value>Service Accounts</value> page",
+    "current_instance": "Accounts do not have access to this instance unless granted under the instance’s",
```

Comment:
This is another case where we should provide the complete statement for translation, but it's currently broken up into separate strings.

Can you follow the example you used for `create_service_account_to_generate_credentials` above to combine the following strings:
* `access_tab`
* `current_instance`

so that we have something like this?:
> `Accounts do not have access to this instance unless granted under the instance’s <value>Access tab</value>.`

**Note** that I moved the period (.) to be after the `</value>` so that it's not included as part of the link.